ocs_ci/ocs/resources/pvc.py

```diff
@@ -181,6 +188,31 @@ def get_attached_pods(self):
                 attached_pods.append(pod_obj)
         return attached_pods
 
+    def create_snapshot(self, snapshot_name=None, wait=False):
+        """
+        Take snapshot of the PVC
+
+        Args:
+            snapshot_name (str): Name to be provided for snapshot
+            wait (bool): True to wait for snapshot to be ready, False otherwise
+
+        Returns:
+            OCS: Kind Snapshot
+
+        """
+        assert self.provisioner in constants.OCS_PROVISIONERS, (
+            "Unknown provisioner"
+        )
+        if self.provisioner == 'openshift-storage.rbd.csi.ceph.com':
+            snap_yaml = constants.CSI_RBD_SNAPSHOT_YAML
+        elif self.provisioner == 'openshift-storage.cephfs.csi.ceph.com':
+            snap_yaml = constants.CSI_CEPHFS_SNAPSHOT_YAML
+        snapshot_name = snapshot_name or f"{self.name}-snapshot-{uuid4().hex}"
```

Comment:
why you do not use the function : helpers.create_unique_resource_name
it do the same things.

Reply:
This is done to make the snapshot name similar to UI default format. It is pvc_name-snapshot. This will help to identify the PVC name from the snapshot name.